unicorecmsunfpa/templates/home.pt

```diff
@@ -5,7 +5,7 @@
 
         <div tal:define="localisation view.get_localisation()" tal:omit-tag="">
             <div id="banner">
-                <a href="/declaration/">
+                <!-- <a href="/declaration/">
```

Comment:
Also, this link is incorrect and should just be removed, making the banner unclickable.